crates/ibc/src/core/ics04_channel/timeout.rs

```diff
@@ -1,5 +1,6 @@
 use core::fmt::{Display, Error as FmtError, Formatter};
 
+#[cfg(feature = "serde")]
```

Comment:
Similarly here, we should put all serde-related items under a submodule, which we mark `#[cfg(feature = "serde")]`

Reply:
https://github.com/cosmos/ibc-rs/pull/331/commits/8cbf8d28f1da05ffb25f7b7b3014edd90865c24e